src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -0,0 +1,354 @@
+import * as React from 'react';
+import { If, Then, Else } from 'react-if';
+import * as _ from 'lodash';
+import {observer} from "mobx-react";
+import {
+    IClinicalGroupMatch, IGenomicGroupMatch, IGenomicMatch, INctTrial, ITrial,
+    ITrialMatch, IDetailedTrialMatch, IArmMatch
+} from "../../../shared/model/MatchMiner";
+import styles from './style/trialMatch.module.scss';
+import { computed } from "mobx";
+import LazyMobXTable from "../../../shared/components/lazyMobXTable/LazyMobXTable";
+import SampleManager from "../sampleManager";
+import DefaultTooltip, { placeArrowBottomLeft } from "../../../shared/components/defaultTooltip/DefaultTooltip";
+
+export type ITrialMatchProps = {
+    sampleManager?: SampleManager | null;
+    trials: Array<ITrial>;
+    trialMatches:Array<ITrialMatch>;
+    nctTrials:Array<INctTrial>;
+    containerWidth: number;
+}
+
+export type ITrialMatchState = {
+    detailedTrialMatches: Array<IDetailedTrialMatch>;
+}
+
+enum ColumnKey {
+    ID = 'ID',
+    TITLE = 'Title',
+    INTERVENTIONS = 'Interventions',
+    MATCHINGCRITERIA = 'Matching Criteria'
+}
+
+class TrialMatchTableComponent extends LazyMobXTable<IDetailedTrialMatch> {
+
+}
+
+@observer
+export default class TrialMatchTable extends React.Component<ITrialMatchProps, ITrialMatchState> {
+
+    constructor(props: ITrialMatchProps) {
+        super(props);
+        if (props.sampleManager) {
```

Comment:
I don't see why the sampleManager should be optional. If it could, then maybe just calculate the detaieldTrialMatches in patientViewStore and pass in.

Reply:
Removed props.sampleManage